.vitepress/components/twitter.vue

```diff
@@ -0,0 +1,16 @@
+
+<template>
+	<blockquote class="twitter-tweet" data-lang="en" data-theme="dark">
+		<p lang="de" dir="ltr">rollkit start <a href="https://t.co/tytjFm1Z4Y">pic.twitter.com/tytjFm1Z4Y</a></p>&mdash; Josh Stein 🤳✨ (@JoshCStein) <a href="https://twitter.com/JoshCStein/status/1783880747301880161?ref_src=twsrc%5Etfw">April 26, 2024</a>
```

Comment:
should this component take a tweet URL as an argument so it can be used for other tweets in the future?

Reply:
also I don't think it's really accomplishing what it should?
<img width="505" alt="Screenshot 2024-05-09 at 3 02 15 PM" src="https://github.com/rollkit/docs/assets/46639943/cab0fc84-f8dc-42c9-9f2d-1d890ce65855">

Reply:
I've tried this, this just takes a minute to load. I think we should have the tweet url as an argument later on when needed.

Reply:
okay i see now, waited around 20 seconds and it showed up after a refresh